pages/docs/reference/compiler-plugins.md

```diff
@@ -117,9 +117,9 @@ plugins {
 The Maven example is similar to the one above.
 
 The plugin specifies the following annotations: 
-[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html), 
-[`@Async`](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/annotation/Async.html), 
-[`@Transactional`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html), 
+[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html);
```

Comment:
There's no need in semicolons where there's a simple comma-separated list of declarations (see how this list is [currently rendered](https://kotlinlang.org/docs/reference/compiler-plugins.html#kotlin-spring-compiler-plugin)).

Reply:
Ya, i got that. Thanks.